quinn-h3/src/frame.rs

```diff
@@ -146,6 +146,13 @@ where
             s.reset(err_code.into());
         }
     }
+
```

Comment:
I feel like this layer isn't adding much value. Can we bypass it by making some inner stuff `pub(crate)` or something?